src/styles/_reset.scss

```diff
@@ -0,0 +1,17 @@
+@import "./utils/vars";
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: $primaryFontFamily;
+}
+
+body {
+  background-color: $secondaryColor;
+  scroll-behavior: smooth;
```

Comment:
it doesn't work on your page
where do you need to apply it so it works?
`html {`